pom.xml

```diff
@@ -338,9 +338,30 @@
           <artifactId>maven-compiler-plugin</artifactId>
           <version>3.8.1</version>
           <configuration>
+            <compilerId>javac-with-errorprone</compilerId>
+            <forceJavacCompilerUse>true</forceJavacCompilerUse>
             <source>1.8</source>
             <target>1.8</target>
+            <compilerArgs combine.children="override">
+              <compilerArg>-Xep:MixedMutabilityReturnType:OFF</compilerArg>
```

Comment:
https://errorprone.info/bugpattern/MixedMutabilityReturnType

It pops up for any method that returns an ArrayList in the general case, but Collections.emptyList() in the empty case. I think we're good, the callers don't try to change the list.